client/src/components/HistoryViewer/HistoryViewer.js

```diff
@@ -157,25 +161,26 @@ class HistoryViewer extends Component {
       schemaUrl,
       VersionDetailComponent,
       compare,
+      compare: { versionFrom = false, versionTo = false },
```

Comment:
I find this destructuring syntax kind of hard to understand what is expected to happen. Is it that you're extracting the `versionFrom` and `versionTo` from `this.props.compare` AND giving them a default value of false? If so, can we do that with `HistoryViewer.defaultProps` instead?

Reply:
Yes it is, and maybe we can. I'll look into it.

Reply:
Sorry refactoring to use `defaultProps` can't be the case, because compare always has a value (since it comes from the store, which has a default value).

It is either false, or the object. `false.versionFrom` is `undefined` which is not the same thing as `false`, and causes issues further down the execution path.